mkdocs.yml

```diff
@@ -3,29 +3,33 @@
 
 # Project information
 site_name: "Frequenz Weather API"
-site_description: "Protobuf spec for the weather API."
+site_description: "Frequenz gRPC API for retrieving weather forecasts."
 site_author: "Frequenz Energy-as-a-Service GmbH"
 copyright: "Copyright © 2023 Frequenz Energy-as-a-Service GmbH"
 repo_name: "frequenz-api-weather"
 repo_url: "https://github.com/frequenz-floss/frequenz-api-weather"
+# TODO(cookiecutter): "main" is the GitHub repo default branch, you might want to update it
+# if the project uses a different default branch.
 edit_uri: "edit/main/docs/"
 strict: true  # Treat warnings as errors
 
 # Build directories
 theme:
   name: "material"
-  logo: logo.png
-  favicon: logo.png
+  # TODO(cookiecutter): You might want to change the logo, the file is located in "docs/"
+  logo: _img/logo.png
+  favicon: _img/logo.png
   language: en
   icon:
     edit: material/file-edit-outline
     repo: fontawesome/brands/github
-  custom_dir: docs/overrides
+  custom_dir: docs/_overrides
```

Comment:
You need to remove the old directory `docs/overrides` too, please have a look at the [updating notes in the releases notes](https://github.com/frequenz-floss/frequenz-repo-config-python/releases) (for all the versions you are jumping too).